### PR TITLE
Use new enhanced logging endpoints

### PR DESCRIPTION
**Pull request by agners**

### CODE REVIEW

File: client/helper.go [hunk 1]
```diff
@@ -167,3 +168,21 @@ func ShowJSONResponse(resp *resty.Response) (success bool) {
...
+			break
+		} else if err != nil {
+			fmt.Println(err)
+			success = false
+			break
+		}
+		fmt.Print(string(p[:n]))
+	}
+	return
+}
```
Comment (mdegat01) [inline client/helper.go:188]: Hm, this isn't working. Testing it now, I don't always get the expected number of results based on range. The API itself seems to be working fine when I hit it `curl` in testing so I believe the problem is in here. Looking into it now.

Comment (mdegat01) [reply client/helper.go]: All good now. Thanks @agners !

**Comment by agners** [comment]
LGTM

File: cmd/host_logs.go [hunk 2]
```diff
@@ -26,24 +27,91 @@ Allowing you to look at the log output generated by the host kernel.
...
 	},
 }
 
 func init() {
+	hostLogsCmd.Flags().BoolP("follow", "f", false, "Continuously print new log entries")
+	hostLogsCmd.Flags().Int32P("lines", "n", 0, "Number of log entries to show")
+	hostLogsCmd.Flags().StringP("identifier", "t", "", "Show entries with the specified syslog identifier")
+	hostLogsCmd.Flags().StringP("boot", "b", "", "Logs of particular boot ID")
+	hostLogsCmd.Flags().Bool("list-identifiers", false, "Show all syslog identifiers")
+	hostLogsCmd.Flags().Bool("list-boots", false, "Show all boot IDs by offset")
```
Comment (pvizeli) [inline cmd/host_logs.go:110]: to be more consistent to our cli maybe we should split that into commands instead flags kinda like
`ha host logs list-boots`

**Comment by mdegat01** [comment]
Note that if testing `ha host logs identifiers` you need to apply [this branch](https://github.com/home-assistant/supervisor/pull/3974) to your system. I realized `/host/logs/identifiers` was not CLI friendly and adjusted it.

**Pull request merged**